cache/manager_test.go

```diff
@@ -1160,7 +1166,7 @@ func TestGetRemote(t *testing.T) {
 	eg, egctx := errgroup.WithContext(ctx)
 	for _, ir := range refs {
 		ir := ir.(*immutableRef)
-		for _, compressionType := range []compression.Type{compression.Uncompressed, compression.Gzip, compression.EStargz} {
+		for _, compressionType := range []compression.Type{compression.Uncompressed, compression.Gzip /*compression.EStargz,*/, compression.Zstd} {
```

Comment:
Is EStargz intentionally omitted here? Could you add a comment line if so?

Reply:
Yes, see https://github.com/moby/buildkit/pull/2344#issuecomment-913921170 and the commit message. Also have "disabled" message in code (not on that line). In current form, it is a release blocker for `v0.10` for me.